static/js/components/TNSATForm.jsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import PropTypes from "prop-types";
+import MenuItem from "@material-ui/core/MenuItem";
+import Select from "@material-ui/core/Select";
+import InputLabel from "@material-ui/core/InputLabel";
+import { makeStyles } from "@material-ui/core/styles";
+import Form from "@rjsf/material-ui";
+
+import { showNotification } from "baselayer/components/Notifications";
+import * as sourceActions from "../ducks/source";
+import * as tnsrobotsActions from "../ducks/tnsrobots";
+
+const useStyles = makeStyles(() => ({
+  chips: {
+    display: "flex",
+    flexWrap: "wrap",
+  },
+  chip: {
+    margin: 2,
+  },
+  tnsrobotSelect: {
+    width: "100%",
+  },
+  tnsrobotSelectItem: {
+    whiteSpace: "break-spaces",
+  },
+  container: {
+    width: "99%",
+    marginBottom: "1rem",
+  },
+}));
+
+const TNSATForm = ({ obj_id }) => {
```

Comment:
What does the AT stand for in TNSATForm?

Reply:
Astronomical transient

Reply:
Should have known that...